### PR TITLE
* Auth Screens Completed - Login, Signup and Get Started

### PR DESCRIPTION
**Pull request by DevSiddharthDesai**
* Code refactoring is pending in which micro level management will be there for components

### CODE REVIEW

File: client/src/App.tsx [hunk 1]
```diff
@@ -1,17 +1,17 @@
 import React from "react";
 import "./App.css";
```
Comment (Akashmalviya) [inline client/src/App.tsx]: are we using this??

File: client/src/components/common/forms/Signin/Signin.tsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+            className="w-full border outline-0 border-inputBorder p-2"
+          />
+        </div>
+        <div className="w-full p-3">
+          <input
+            type="password"
+            placeholder="Password"
+            className="w-full border outline-0 border-inputBorder p-2"
+          />
+        </div>
```
Comment (Akashmalviya) [inline client/src/components/common/forms/Signin/Signin.tsx]: can we create new Input component, and use that

File: client/src/components/common/forms/Signin/Signin.tsx [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+        <div className="w-full p-3">
+          <button className="bg-buttonPrimary w-full px-4 py-4 mb-4">
+            <Link to={"#"} style={{ width: "100%", display: "block" }}>
+              Get Started
+            </Link>
+          </button>
```
Comment (Akashmalviya) [inline client/src/components/common/forms/Signin/Signin.tsx]: create Button component, which can accept but type like primary, secondly, outline.

File: client/src/components/common/forms/Signup/Signup.tsx [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+            className="w-full border outline-0 border-inputBorder p-2"
+          />
+        </div>
+        <div className="w-full p-3">
+          <input
+            type="password"
+            placeholder="Password"
+            className="w-full border outline-0 border-inputBorder p-2"
+          />
+        </div>
```
Comment (Akashmalviya) [inline client/src/components/common/forms/Signup/Signup.tsx]: Use Input component

File: client/src/components/common/forms/Signup/Signup.tsx [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+        <div className="w-full p-3">
+          <button className="bg-buttonPrimary w-full px-4 py-4 mb-4">
+            <Link to={"#"} style={{ width: "100%", display: "block" }}>
+              Get Started
+            </Link>
+          </button>
```
Comment (Akashmalviya) [inline client/src/components/common/forms/Signup/Signup.tsx]: use Button component

File: client/src/index.css [hunk 6]
```diff
@@ -7,22 +7,22 @@ body {
...
-  background: linear-gradient(to bottom, yellow 50%, white 50%);
-  background-size: 100% 400px;
-  background-repeat: no-repeat;
-  position: relative;
-  overflow: hidden;
+  /* background: linear-gradient(to bottom, yellow 50%, white 50%); */
+  /* backg/round-size: 100% 400px; */
+  /* background-repeat: no-repeat; */
+  /* position: relative; */
+  /* overflow: hidden; */
```
Comment (Akashmalviya) [inline client/src/index.css]: remove commented code

File: client/src/pages/auth/Auth.tsx [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+            </p>
+            <button className="bg-buttonPrimary px-4 py-4 mb-4">
+              <Link
+                onClick={onRegisterClickHandler}
+                to={"#"}
+                style={{ width: "100%", display: "block" }}
+              >
+                Get Started
+              </Link>
+            </button>
```
Comment (Akashmalviya) [inline client/src/pages/auth/Auth.tsx]: use Button components

File: client/src/App.tsx (hunk 1 above)
Comment (DevSiddharthDesai) [reply client/src/App.tsx]: No, It's not useful as of now. So, I will remove it for now.

File: client/src/components/common/forms/Signin/Signin.tsx (hunk 2 above)
Comment (DevSiddharthDesai) [reply client/src/components/common/forms/Signin/Signin.tsx]: Yes. Working on it. Noted!

File: client/src/components/common/forms/Signin/Signin.tsx (hunk 3 above)
Comment (DevSiddharthDesai) [reply client/src/components/common/forms/Signin/Signin.tsx]: Noted.

File: client/src/components/common/forms/Signup/Signup.tsx (hunk 5 above)
Comment (DevSiddharthDesai) [reply client/src/components/common/forms/Signup/Signup.tsx]: Noted

File: client/src/components/common/forms/Signup/Signup.tsx (hunk 4 above)
Comment (DevSiddharthDesai) [reply client/src/components/common/forms/Signup/Signup.tsx]: Noted

File: client/src/pages/auth/Auth.tsx (hunk 7 above)
Comment (DevSiddharthDesai) [reply client/src/pages/auth/Auth.tsx]: Noted!

File: client/src/components/common/forms/Signup/Signup.tsx (hunk 5 above)
Comment (DevSiddharthDesai) [reply client/src/components/common/forms/Signup/Signup.tsx]: Completed

**Pull request merged**